update(getting started): fill out the todo, fix first page (#1623)
bonitasoft/bonita-doc: modules/ROOT/pages/getting-started-index.adoc

= Getting started 
:description: This is the fastest way to learn, try, and master all concepts of a Living Application.

To get started with Bonita, learn how to create your first complete living application.

[.card-section]
== Prerequisite

[.card.card-index]
--
xref:bonita-studio-download-installation.adoc[[.card-title]#Bonita Studio Installation# [.card-body.card-content-overflow]#pass:q[Bonita's principal tool to create the main elements of all your projects]#]
--

[.card-section]
== The 9-step hands-on journey to your first Bonita Living Application

[.card.card-index]
--

> xref:draw-bpmn-diagram.adoc[[.card-title]#Draw a BPMN diagram# [.card-body.card-content-overflow]#pass:q[Process automation, "most of the time" ;), starts with a process]#]

Reviewer: it feels weird to have an ascii smiley here. Ascii smiley have to be put at the end of a sentence, else it's complicated to read. 
Moreover, I think that _"Most of the time" ;)_ is inadequate here. Keep the sentence simple, it's a card.

```suggestion
xref:draw-bpmn-diagram.adoc[[.card-title]#Draw a BPMN diagram# [.card-body.card-content-overflow]#pass:q[Process automation usually starts with a process]#]
```

Author: They always start with a process. So, staying very plain, this is what I will say.

> xref:draw-bpmn-diagram.adoc[[.card-title]#Draw a BPMN diagram# [.card-body.card-content-overflow]#pass:q[Process automation, "most of the time" ;), starts with a process]#]

Author: ```suggestion
xref:draw-bpmn-diagram.adoc[[.card-title]#Draw a BPMN diagram# [.card-body.card-content-overflow]#pass:q[Process automation always starts with a process]#]
```

--

[.card.card-index]
--
xref:define-business-data-model.adoc[[.card-title]#Define a Business Data Model (BDM)# [.card-body.card-content-overflow]#pass:q[The data model represents the business data handled by processes.]#]
--

[.card.card-index]
--
xref:declare-business-variables.adoc[[.card-title]#Declare business variables# [.card-body.card-content-overflow]#pass:q[Learn hot to link business data and processes]#]
--

[.card.card-index]
--

> xref:declare-contracts.adoc[[.card-title]#Declare contracts# [.card-body.card-content-overflow]#pass:q[This step defines how the UI and the process interact]#]

Author: ```suggestion
xref:declare-contracts.adoc[[.card-title]#Declare contracts# [.card-body.card-content-overflow]#pass:q[Contracts account for the interfaces allowing processes and UI to interact]#]
```

--

[.card.card-index]
--
xref:create-web-user-interfaces.adoc[[.card-title]#Create forms# [.card-body.card-content-overflow]#pass:q[Forms use data to feed processes through contracts]#]
--

[.card.card-index]
--
xref:define-who-can-do-what.adoc[[.card-title]#Define who can do what# [.card-body.card-content-overflow]#pass:q[Map organization entities to each human task]#]
--

[.card.card-index]
--
xref:configure-email-connector.adoc[[.card-title]#Configure an email connector# [.card-body.card-content-overflow]#pass:q[Most of the processes need email notifications, right?]#]
--

[.card.card-index]
--

> xref:design-application-page.adoc[[.card-title]#Design an application page# [.card-body.card-content-overflow]#pass:q[Unlike the form, an application page displays more than the information of a human task]#]

Author: ```suggestion
xref:design-application-page.adoc[[.card-title]#Design an application page# [.card-body.card-content-overflow]#pass:q[Application pages information is more generic than human tasks information]#]
```

> This is the fastest way to learn, try, and master all concepts of a Living Application.

Reviewer: ```suggestion
To get started with Bonita, learn how to create your first complete living application.
```

[.card-section]
== Prerequisite

[.card.card-index]
--

> xref:bonita-studio-download-installation.adoc[[.card-title]#Bonita Studio Installation# [.card-body.card-content-overflow]#pass:q[Bonita's main tool to create the main elements of all your projects]#]

Reviewer: You use 'main' twice in a short sentence, it's better to use a synonym :)

```suggestion
xref:bonita-studio-download-installation.adoc[[.card-title]#Bonita Studio Installation# [.card-body.card-content-overflow]#pass:q[Bonita's principal tool to create the main elements of all your projects]#]
```

--

[.card-section]
== The 9-step hands-on journey to your first Bonita Living Application

[.card.card-index]
--
xref:draw-bpmn-diagram.adoc[[.card-title]#Draw a BPMN diagram# [.card-body.card-content-overflow]#pass:q[Process automation, "most of the time", starts with a process]#]
--

[.card.card-index]
--

> xref:define-business-data-model.adoc[[.card-title]#Define a Business Data Model (BDM)# [.card-body.card-content-overflow]#pass:q[The business data will be handled by the process. First, you need to create the data model]#]

Reviewer: This sentence is too long, the last words are cut in the card.

```suggestion
xref:define-business-data-model.adoc[[.card-title]#Define a Business Data Model (BDM)# [.card-body.card-content-overflow]#pass:q[The data model represents the business data handled by processes.]#]
```

--

[.card.card-index]
--

> xref:declare-business-variables.adoc[[.card-title]#Declare business variables# [.card-body.card-content-overflow]#pass:q[This is the step to link the business data to the process]#]

Reviewer: ```suggestion
xref:declare-business-variables.adoc[[.card-title]#Declare business variables# [.card-body.card-content-overflow]#pass:q[Learn hot to link business data and processes]#]
```

--

[.card.card-index]
--

> xref:declare-contracts.adoc[[.card-title]#Declare contracts# [.card-body.card-content-overflow]#pass:q[This step defines how the UI and the process interact]#]

Reviewer: ```suggestion
xref:declare-contracts.adoc[[.card-title]#Declare contracts# [.card-body.card-content-overflow]#pass:q[Contracts are the interfaces allowing processes and UI's to interact]#]
```

Author: Not UI's. Je commit avec ce petit ajustement

--

[.card.card-index]
--

> xref:create-web-user-interfaces.adoc[[.card-title]#Create forms# [.card-body.card-content-overflow]#pass:q[The forms use the data to feed the process, through the contracts]#]

Reviewer: ```suggestion
xref:create-web-user-interfaces.adoc[[.card-title]#Create forms# [.card-body.card-content-overflow]#pass:q[Forms use data to feed processes through contracts]#]
```

--

[.card.card-index]
--
xref:define-who-can-do-what.adoc[[.card-title]#Define who can do what# [.card-body.card-content-overflow]#pass:q[Map organization entities to each human task]#]
--

[.card.card-index]
--
xref:configure-email-connector.adoc[[.card-title]#Configure an email connector# [.card-body.card-content-overflow]#pass:q[Most of the processes need email notifications, right?]#]
--

[.card.card-index]
--

> xref:design-application-page.adoc[[.card-title]#Design an application page# [.card-body.card-content-overflow]#pass:q[Unlike the form, an application page displays more than the information of a human task]#]

Reviewer: This sentence is too long for the card.

```suggestion
xref:design-application-page.adoc[[.card-title]#Design an application page# [.card-body.card-content-overflow]#pass:q[An application page is not bounded to a human task, unlike a form]#]
```

Author: je rectifie l'anglais, je revois la phrase et je commit

--

[.card.card-index]
--

> xref:create-application.adoc[[.card-title]#Create an application# [.card-body.card-content-overflow]#pass:q[And this is where everything is put together to build a rich and valuable end-user experience]#]

Reviewer: This sentence is too long for the card.

```suggestion
xref:create-application.adoc[[.card-title]#Create an application# [.card-body.card-content-overflow]#pass:q[This is where everything is put together to build a rich, valuable end-user experience]#]
```
